docs/docs/starlark-reference/read-file.md

```diff
@@ -3,7 +3,7 @@ title: read_file
 sidebar_label: read_file
 ---
 
-The `read_file` function reads the contents of a file specified by the given [locator][locators-reference], and requires that the Starlark script is part of a [package][packages-reference]. `read_file` executes [at interpretation time][multi-phase-runs-reference] so the file contents won't be displayed in the preview.
+The `read_file` function reads the contents of a file specified by the given [locator][locators-reference] and executes [at interpretation time][multi-phase-runs-reference] so the file contents won't be displayed in the preview. This instruction returns the content of the file in a string type. Please note that the files being read from must themselves be part of a Kurtosis package, as explained [here](../concepts-reference/locators.md#important-package-restriction).
```

Comment:
"so the file contents won't be displayed in the preview" was confusing to me; it seems to suppose the user already knows what preview is, are confused, and is guarding against confused users. That doesn't feel right for this doc, which I would expect to first be confused by curious people.